src/main/java/uk/gov/hmcts/reform/professionalapi/domain/Organisation.java

```diff
@@ -173,4 +183,8 @@ public void setOrganisationIdentifier(String organisationIdentifier) {
     public boolean isOrganisationStatusActive() {
         return OrganisationStatus.ACTIVE == getStatus();
     }
+
+    public OrganisationMfaStatus getOrganisationMfaStatus() {
+        return organisationMfaStatus;
+    }
```

Comment:
Do we need an explicit getter and setter? Can we try with @Getter and @Setter annotations?

Reply:
The @getter and @setter annotations are already present in the Organisation class, however all the fields have explicit getters, so to stay consistent I've kept the explicit getter. But obviously it can be removed.